### ISSUE TITLE
Cartoon trace tilt issue

### ISSUE THREAD

**Issue by giagitom** [molstar/molstar#649]
Sometimes it happens to note strange tilts like this on cartoon trace. This is entry code 4V6X
![image](https://user-images.githubusercontent.com/20627977/206928599-1657cf60-117d-46d4-a987-4a4560f02e47.png)

**Comment by arose** [comment]
Yeah, that can happen for tight curves with how the curve normals are calculated.

**Comment by giagitom** [comment]
It seems like a complete 360° inversion
